src/driver/postgres/PostgresDriver.ts

```diff
@@ -1101,7 +1101,8 @@ export class PostgresDriver implements Driver {
             database: credentials.database,
             port: credentials.port,
             ssl: credentials.ssl,
-            connectionTimeoutMillis: options.connectTimeoutMS
+            connectionTimeoutMillis: options.connectTimeoutMS,
+            application_name: options.applicationName || "typeorm app"
```

Comment:
Can this default to undefined instead

Reply:
sure, my thought was: if any typeorm app is visible with "typeorm app" it'll help to distinguish between typeorm apps and all other apps in the database overview. A bit more "clean by default". But i see your point...